### DISCUSSION TITLE
Matomo Tag Manager Example as with Google Tag Manager

### DISCUSSION THREAD

**Discussion by csaeum** [orestbida/cookieconsent#498] [General]
In your demo you have an example where the consent is sent to the GTM or to the DataLayer.

BSP:
`typeof gtag === 'function' && gtag('consent', 'update', {
                'technisch': 'active'
            });
`
Now my question, what does this have to look like for the Matomo Tag Manager?
I use both Tag Managers, the main part goes to the Matomo Tag Manager and everything else via Google.

Shouldn't this be sent as an event in order to be able to access or react to both directly?

**Comment by csaeum** [comment]
ok could solve it now.

Both arrive in the DataLayer in GTM and MTM.
```javascript
if (cc.allowedCategory('google_analytics')) {
    _mtm.push({
        'event': 'consent_google',
        'consent_google': 'active'
    });
    dataLayer.push({
        'event': 'consent_google',
        'consent_google': 'active'
    });
} else {
    _mtm.push({
        'event': 'consent_google',
        'consent_google': 'denied'
    });
    dataLayer.push({
        'event': 'consent_google',
        'consent_google': 'denied'
    });
}
```
But if you have a better solution, I'm open to it!

**Reply by orestbida** [reply]
This is probably how I would also do it!
